src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -60,19 +45,29 @@ fun interface EmailService {
   fun send(email: Email): Result<UUID>
 }
 
-abstract class Email {
-  abstract val address: String
-  abstract val personalisation: Map<String, String?>
+abstract class Email(
+  val address: String,
+  prisonerFirstName: String,
+  prisonerLastName: String,
+  prisonerNumber: String,
+  date: LocalDate = LocalDate.now(),
+  comments: String? = "",
+) {
+  private val common = mapOf(
```

Comment:
These are some of the common attributes across all of the templates.

I have probably missed some but this should manifest itself when we introduce support for other emails/templates.